ngraph/frontend/tensorflow/src/op/arg_min_max.cpp

```diff
@@ -14,46 +14,43 @@ namespace tf {
 namespace op {
 
 OutputVector TranslateArgMinMax(const NodeContext& node, std::string mode) {
-    Output<Node> ng_input = node.get_ng_input(0);
+    Output<Node> ng_input = node.get_input(0);
 
     std::vector<int64_t> tf_dim;
-    GetStaticInputVector(node, 1, &tf_dim);
+    get_static_input_vec(node, 1, &tf_dim);
 
     Shape input_shape = ng_input.get_shape();
     size_t input_rank = input_shape.size();
 
-    if (tf_dim.size() != 1) {
-        throw errors::InvalidArgument("ArgMax Op: dimension must be scalar, operates on a single axis");
-    }
+    TF_OP_VALIDATION_CHECK(node, tf_dim.size() == 1, "ArgMax Op: dimension must be scalar, operates on a single axis");
 
     // If input dimension is negative, make it positive
     if (tf_dim[0] < 0) {
-        NGRAPH_VLOG(3) << "Input dimension is negative, make it positive " << tf_dim[0];
+        NGRAPH_DEBUG << "Input dimension is negative, make it positive " << tf_dim[0];
         tf_dim[0] = (int64_t)input_rank + tf_dim[0];
     }
-    NGRAPH_VLOG(3) << "Axis along which to compute " << tf_dim[0];
+    NGRAPH_DEBUG << "Axis along which to compute " << tf_dim[0];
     size_t k_axis = tf_dim[0];
 
     auto ng_et = node.get_attribute<element::Type>("output_type");
 
-    auto ng_k = ConstructNgNode<Constant>(node.get_name(), element::i64, Shape{}, std::vector<int64_t>({1}));
+    auto ng_k = make_shared<Constant>(element::i64, Shape{}, std::vector<int64_t>({1}));
 
     std::string sort = "none";
     auto ng_topk = std::make_shared<TopK>(ng_input, ng_k, k_axis, mode, sort, ng_et);
```

Comment:
propose to use `TopK` of exact opset and specify opset explicitly. 'opset8::TopK'

Reply:
we use "using namespace opset8" in this file